docs/ArgoCD Integration.md

```diff
@@ -310,8 +310,8 @@ data:
 The [ArgoCD Server Helm chart](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd) supports defining `argocd-cm` settings through [values.yaml](https://github.com/argoproj/argo-helm/blob/6ff050f6f57edda1e6912ef0bb17d085684e103e/charts/argo-cd/values.yaml#L1155-L1157):
 
```

Comment:
Could you please add a version here? Other readers can detect fasted, if the docs are outdated...

```suggestion

The configuration below is tested with helm chart version x.y.z

```

Thanks